packages/expo-audio/src/AudioModule.types.ts

```diff
@@ -11,18 +11,19 @@ import {
   RecordingStatus,
 } from './Audio.types';
 
-export interface AudioModule {
+/**
+ * @hidden
+ */
+export declare class AudioModuleDefinition {
```

Comment:
I would be more in favor of naming native module classes as `NativeAudioModule` which is very clear that it refers to the object exported from native.

Reply:
Done!